src/text_cleaning/utils.py

```diff
@@ -0,0 +1,75 @@
+__all__ = ['count_words', 'count_punctuation', 'count_how_many_words_are_repeating', 'count_misspelled_words']
+
+import re
+from collections import Counter
+from string import punctuation
+
+from src.text_cleaning.spell_checker import SmartSpellChecker
+
+underscores_to_replace = {
+    'Generic_Name': 'name',
+    'OTHER_NAME': 'name',
+    'STUDENT_NAME': 'name',
+    'Generic_Namea': 'name',
+    'PROPER_NAME': 'proper name',
+    'PROEPR_NAME': 'proper name ',
+    'Generic_School': 'school',
+    'SCHOOL_NAME': 'school',
+    'Generic_school': 'school',
+    'TEACHER_NAME': 'teacher',
+    'Generic_City': 'city',
+    'LOCATION_NAME': 'location',
+    'STORE_NAME': 'store',
+    'RESTAURANT_NAME': 'restaurant',
+    'LANGUAGE_NAME': 'language',
+}
+
+
+def preprocess_test(text: str):
```

Comment:
Please add annotation to return values for all functions here, like `def function_name(...) -> List[float]`

Reply:
Yep, I've modified that